lib/muffin/frostings/attributes.rb

```diff
@@ -63,5 +63,9 @@ class NestedAttribute
     def initialize(attributes)
       self.attributes = attributes
     end
+
+    def persisted?
+      attributes[:id].present?
+    end
```

Comment:
It's indeed needed twice. Once for the `Muffin::Base` class and for each `NestedAttribute`, too. If you leave it out for the `NestedAttribute`, `fields_for` fails.